### ISSUE TITLE
Implement Python like language

### ISSUE THREAD

**Issue by LordStephen77** [egordorichev/neko8#30]
Neko8 is very nice but I can not use it because it uses asm?!? is a tool for hobbyists, passionate and nostalgic, asm is out of measure for a human being. I've been hobbies for many years, but I think it's a good idea to implement a scripting language that is very simple, everyone's reach. Especially trying to simplify the APIs, PICO-8, TIC-80 and LIKO-12 have a big defect... they use LUA as a script language, and it is not the best choice in my opinion. Python is perfect! it would be nice to implement it similarly to Godot Game Engine's GDScript, a middle ground between Python and the old QuickBasic makes Neko8 perfect and increases the spread of the project. Many people want to use these PICO-8 Like Fantasy Consoles, but they give up because of the complexity and lack of proper tutorials and documentation.

Neko8 implements the documentation inside it, this is a nice feature.

Your work is extraordinary, you can make Neko8 the absolute reference for fans, students and all those people who want to learn calmly, and make a project in no time without frustration.

Thanks for your nice job.

**Comment by incinirate** [comment]
Actually to my knowledge, neko8 implements Lua, with ASM being interpreted within Lua.

**Comment by LordStephen77** [comment]
@incinirate I compiled the source 1 hour ago, i tested helloword_asm. Now I try to do something and see how she uses LUA.

Thank you for your response, I really hope that my idea is taken into account. The project has great potential, a language like Python is much easier and recalls the 80's ;)

**Comment by RamiLego4Game** [comment]
Let's start a Lua vs Python war: Hey, LuaJIT is a lot better than Python, a code file would take 100 light years to compile, while LuaJIT compress the code in 0.0001 seconds.

**Comment by egordorichev** [comment]
Yup, neko8 supports lua as main scripting language, moonscript, asm and BASIC. To switch between them use `new 'lang'`. `new` defaults to lua, `new asm` creates asm cart.

**Comment by egordorichev** [comment]
And you should use lua build for now ;) 
https://github.com/egordorichev/neko8/blob/master/Game%20Jam%20Special%20Edition/GJSE.love

To run it, you will need love2d. Run it with:

```
love ./GJSE.love
```

**Comment by pi-pi3** [comment]
Hi there, I'm the developer of `asm.lua`. `asm.lua` isn't being supported anymore. It was a quick experiment that is now in stagnation and it won't be supported in `neko8` anymore. Lua remains the default in neko. But there have been ideas of implementing other scripting languages, like JavaScript and a BASIC. If anyone is willing to implement Python, it shouldn't be too hard, now that a C++ port is being worked on

**Comment by egordorichev** [comment]
@pi-pi3 did not you want to keep it? It is awesome, yet simple. Porting it to c++ (or implementing lua sandbox for it) shouldn't be hard. As we all know, I'm working on a few jams at the moment, that's why I did not make a single commit in the last few weeks. But the project is NOT dead.

**Comment by pi-pi3** [comment]
I'm fine with you working on it, but I won't be anymore

**Comment by egordorichev** [comment]
Ok, that's fine ;)

**Comment by LordStephen77** [comment]
@RamiLego4Game Liko-12 is very cool <3 the font changes because the standard one can read only an alien with 12 eyes :D

Thanks for the gentle response guys. The new C ++ port is a nice idea, I guess I can see new features and improved performance in the future. Thanks for your great work.
